src/app/app.component.html

```diff
@@ -1,33 +1,16 @@
 <app-header>
   <div class="header-right-content">
-    <app-button [buttonText]="buttonLabelService.getLabel('LOGIN')"></app-button>
+    <app-button
+      [buttonText]="
+        buttonLabelService.getLabel(
+          authService.isAuthorised ? 'LOGOUT' : 'LOGIN'
+        )
+      "
+      (clicked)="onLogoutClick()"
```

Comment:
If I am currently not logged in, the UI shows a 'login' button however it is not doing anything. Maybe the task description should be a little more specific on this...
If you want, you can address it like adding some 'welcome text' or focus on the login form once the 'login' button is clicked. It is up to you.